### ISSUE TITLE
The required props is not present if the answer is mandatory

### ISSUE THREAD

**Issue by jdirand** [InseeFr/Lunatic#715]
Currently, the required value is never set even if the answer is mandatory.
For accessibility purposes, if the user's answer is mandatory, then the html following html elements should have the "required" attribute set to true : select, input, textarea
